src/components/Scrollbar.tsx

```diff
@@ -1,115 +1,38 @@
   fade = 'both',
-  parentHeight = 0,
-  bottomPadding = 0,
-  titlePadding = 0,
 }: ChildrenProp & ScrollbarProps) {
-  const { height = 0, ref } = useResizeDetector({ handleWidth: false })
-  const maximumHeightSize = 370 // the maximum height of the block by design
-  const fullHeightInPercentage = 100 // the thumb height in percentages
-
-  const thumbRef = useRef() as MutableRef<HTMLDivElement>
-  const [thumbHeight, setThumbHeight] = useState(fullHeightInPercentage)
-  const extaBottomPadding = parentHeight > maximumHeightSize ? bottomPadding : 0
-  const extaTitlePadding = titlePadding
-  const extraReservedSpace = extaBottomPadding + extaTitlePadding
-
-  const { overflows, scrollMaxHeight, isOnTop, isOnBottom, wrapperRef } =
-    useIsOverflow(ref, height - extraReservedSpace)
-
-  const handleScroll = () => {
-    if (!thumbRef.current || !wrapperRef.current) return
-    const wrapCurrent = wrapperRef.current
-
-    // .scrollTop counts whole box, not only visible
-    // we should divide whole box by the visible to get how many visible boxes are in scrollable container
-    const numberOfViews = wrapCurrent.scrollHeight / wrapCurrent.clientHeight
-    const scroll = wrapCurrent.scrollTop / numberOfViews
-
-    // to prevent floating-point operation use px here
-    thumbRef.current.style.top = scroll + 'px'
-  }
-
-  // Listens if something inside the wrapBox has changed
-  useEffect(() => {
-    const { current } = wrapperRef
-    if (!current) return
-
-    setTimeout(() => {
-      const numberOfViews = current.scrollHeight / current.clientHeight
-      setThumbHeight(fullHeightInPercentage / numberOfViews)
-    }, 400)
-  }, [
-    titlePadding,
-    parentHeight,
-    bottomPadding,
-    ref,
-    wrapperRef,
-    wrapperRef.current?.scrollHeight,
-    wrapperRef.current?.clientHeight,
-  ])
+  const hasWebKit = useHasWebkit()
+  const { isOverflow, wrapperRef } = useIsOverflow()
 
   return (
-    <div ref={ref} className={scrollContainer}>
-      <div
-        ref={wrapperRef}
-        className={classNamesToString(
-          overflow('overflow-auto'),
-          'scrollbar-hide'
-        )}
-        style={{
-          maxHeight: scrollMaxHeight,
-          marginRight: overflows ? '1rem' : undefined,
-        }}
-        onScroll={handleScroll}
-      >
-        {overflows && (
-          <div className={classNamesToString('custom-scrollbar-body')}>
-            <div
-              ref={thumbRef}
-              className={classNamesToString('custom-scrollbar-thumb')}
-              style={{ height: thumbHeight + '%' }}
-            />
-          </div>
-        )}
-        {isOnTop && (fade === 'both' || fade === 'top') && <Fade />}
-        {children}
-        {isOnBottom && (fade === 'both' || fade === 'bottom') && (
-          <Fade bottom />
-        )}
-      </div>
+    <div
+      ref={wrapperRef}
+      className={wrapperStyles}
+      style={{
+        marginRight: isOverflow && hasWebKit ? '-0.7rem' : '0rem',
```

Comment:
Basically, we needed the `maxHeight` only for `<Fade />`, because it was `absolute`, aligned to `outerDiv` (not this one). This `innerDiv` didn't know about the `<Card />` wrapper component and tried to overflow using `outerDiv`, but it wasn't configured to overflow properly (I am not sure, if we can make `parent` overflow if the children overflows 😄)